inst/compareScenarios/cs_02_energyDemand.Rmd

```diff
@@ -0,0 +1,51 @@
+# Final energy demand
+
+## All buildings
+
+## Residential
+
+### Space heating
+
+```{r (en)-Residential-Space heating-(carrier)}
+for (en in c("FE", "UE")) {
+  tot <- paste0(en, "|Residential|Space heating")
+  vars <- paste(tot, carrier, sep = "|")
+  showAreaAndBarPlots(data, vars = vars, tot = tot, orderVars = "user", scales = "fixed")
+  walk(vars, showLinePlots, data = data)
```

Comment:
I'm a bit confused how the script (and the reader) knows that this is the purrr function ```walk``` (I guess)? Maybe because purrr just happens to be loaded by some other package? For PR #4  I had to add purrr as an imported package anyway, so you can probably safely write ```purrr::walk``` here and elsewhere.

Reply:
Thanks Ricarda for pointing this out. I'm really not sure how to handle dependencies from the scripts properly. I decided to load packages with explicit lists of functions in `inst/compareScenarios/preprocessing.R` for compareScenarios.